apps/wallet-dashboard/components/Dialogs/SendToken/views/SentSuccessView.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { useGetTransaction } from '@iota/core';
+import { InfoBoxType, InfoBox, InfoBoxStyle } from '@iota/apps-ui-kit';
+import { Warning } from '@iota/ui-icons';
+import { getExtendedTransaction } from '@/lib/utils';
+import { useCurrentAccount } from '@iota/dapp-kit';
+import { TransactionDialogDetails } from '../../transaction';
+
+interface SentSuccessProps {
+    digest?: string;
+    onClose: () => void;
+}
+
+export function SentSuccessView({ digest, onClose }: SentSuccessProps) {
+    const currentAccount = useCurrentAccount();
+    const { data, isError, error } = useGetTransaction(digest || '');
+
+    const transaction = data && getExtendedTransaction(data, currentAccount?.address || '');
+
+    if (isError) {
```

Comment:
we can make this before the previous line. I don't think it's necessary to `getExtendedTransaction` if there has been an error